### ISSUE TITLE
Add `name` and `description` to `PaymentMethod`

### ISSUE THREAD

**Issue by mistermoe** [TBD54566975/tbdex#261]
[PaymentMethod](https://github.com/TBD54566975/tbdex/tree/main/specs/protocol#paymentmethod) is the type used for the elements within `Offering.payinMethods[]` and `Offering.payoutMethods[]`. `kind` is reflected as an enum which often won't be cased or spaced in a manner that would be rendered on screen. 

Proposing that we add `name` (or `title`) and `description` to `PaymentMethod` to provide client apps with something they can take an render as is. instinctively thinking both should be optional.

one example where description could be helpful is supported btc address types

**Comment by selfissued** [comment]
Again, per https://github.com/TBD54566975/tbdex/issues/264#issuecomment-1989681165 , internationalization of text strings intended for display to humans is a prime concern.

I agree that `kind` should be treated as a unique identifier, and not displayed in the UI.

Have you considered an approach where the is a publicly-available table of (`kind`, `language`, "Description in that language") rows that can be used to drive the UI, rather than embedding these strings in the protocol?

**Comment by mistermoe** [comment]
yep definitely considered the publicly available table approach. I think over time, that will certainly be helpful as conventions begin to emerge, but until and if the protocol reaches some level of maturity, i imagine there will be many novel `kind`s with each new PFI. 

I'm hesitant to say something along the lines of "`kind` must be one of the options in said table" right out of the gate because of a hunch that it'll create a sense of paralysis for PFIs when they get to the point of creating their first offering. Imagining it'll be something like:
1. Developer arrives at specifying `kind`. Looks at spec text to figure out what `kind` is. 
2. Reads that `kind` value must be one of the options in a table in the spec. 
3. Looks at table and realizes that `CHEESE_CURD` is not a `kind` in the table
4. Does not feel confident or committed enough to submit a PR to add `CHEESE_CURD` as an acceptable `kind`
5. Feels stuck

I do think it'd be worthwhile to include spec text that conveys our intent to establish a registry of _common_ `kind`s and language-specific descriptions incl. a process around adding new `kind`s.
